src/main/java/com/uid2/client/IdentityInput.java

```diff
@@ -0,0 +1,93 @@
+package com.uid2.client;
+
+import com.google.gson.JsonObject;
+
+import java.security.MessageDigest;
+
+public class IdentityInput {
+    static public IdentityInput fromEmail(String email) {
```

Comment:
Should we follow conventions? https://stackoverflow.com/questions/27885860/what-is-difference-between-public-static-and-static-public

Reply:
Yep. Let me take a quick look to see the best way to pick these issues up at build time